apps/settings/js/panels/app_storage/panel.js

```diff
@@ -48,15 +52,18 @@ define(function(require) {
       },
 
       onBeforeShow: function() {
+        window.addEventListener('panelready', _forceUpdate);
```

Comment:
You can call `AppStorage.updateInfo` directly without waiting for the `panelready` event because `onBeforeShow` means the panel is about to be displayed.